ignite/distributed/comp_models/base.py

```diff
@@ -269,7 +271,9 @@ def broadcast(
         return tensor
 
     @abstractmethod
-    def _do_all_reduce(self, tensor: torch.Tensor, op: str = "SUM") -> torch.Tensor:
+    def _do_all_reduce(
+        self, tensor: torch.Tensor, op: str = "SUM", group: Optional[Union[Any, List[int]]] = None
```

Comment:
Check `group` signature here as well, should it be `Optional[Union[Any, List[int]]]` ?